doc/sign-package.md

```diff
@@ -79,44 +81,41 @@ Included is a sample JSON configuration where the elements will be described nex
-EnvironmentVariables | Enclave defined environment variables
-HostEnvironmentVariables | A list of environment variables that can be imported from the insecure host
-Hostname | The default hostname exposed to application
-CurrentWorkingDirectory | The default working directory for the application
-ForkMode | Specify the mode used for the experimental pseudo fork feature. Refer to [doc/design/fork.md](doc/design/fork.md) for more details. The default mode is `none`, which disables the feature.
-Mount | Set if parameters for informing Mystikos to automatically mount a set of directories or ext2 disk images from the host into the TEE. Refer to [doc/design/mount-config-design.md](doc/design/mount-config-design.md) for more details. By default no extra mounts are added to the root filesystem.
-UnhandledSyscallEnosys | This option would prevent the termination of a program using myst_panic when an unimplemented syscall is encountered in the mystikos kernel. The default value is `false`, which implies that we terminate on unhandled syscalls by default. If `true`, it will cause the syscall to return ENOSYS error.
+Settings | Type | Description
+-|-|-
+~~UserMemSize~~ | `int \| string` | Deprecated, use `MemorySize` instead
+MemorySize | `int \| string` | Amount of memory your application needs to run. Try not to make this just a very large number as the larger this number needs to be the slower load time will be. Value can be bytes (just a number), **k**ilobytes (for example `"128k"`), **m**egabytes (for example `"512m"`), or **g**igabytes (for example `"1g"`)
+MainStackSize | `int \| string` | Stack size of your application's main process. Defaults to 1536k (or 1.5M) bytes. Normally, you do not need to customize this. If running an application generates a OOM error like in [#612](https://github.com/deislabs/mystikos/issues/612), try tuning this value, e.g. to 8M. Value can be bytes (just a number), **k**ilobytes (for example `"128k"`), **m**egabytes (for example `"512m"`), or **g**igabytes (for example `"1g"`)
+MaxAffinityCPUs | `int` | This setting limits the number of CPUs reported by sched_getaffinity()
+NoBrk | `boolean \| int` | If set to true(or 1), brk syscall returns -ENOTSUP. Defaults to `false`. Set this to true for program involves multi-threading.
+ApplicationPath | `string` | The executable path relative to the root of your appdir. This executable name is also used to determine the final application name once packaged.
+HostApplicationParameters | `boolean \| int` | This parameter specifies if application parameters can be specified on the command line or not. If true, the command line arguments are used instead of the ApplicationParameters
+ApplicationParameters | `[string] \| string` | Enclave defined application parameters if HostApplicationParameters is set to false.
```

Comment:
` if HostApplicationParameters is set to false` -> `, effective if HostApplicationParameters is set to false`